data/tests/remotes.d/testing.conf

```diff
@@ -1,3 +1,4 @@
 [fwupd Remote]
 Enabled=true
 MetadataURI=file:///tmp/fwupd-self-test/testing.xml
+ApprovalRequired=true
```

Comment:
It might be worth adding a `ApprovalRequired=false` to all other remotes to at least demonstrate that this option is turned off by default for them.

Reply:
Good idea, fixed.